### PR TITLE
Skip PersistentPreRun on version subcmd

### LINKED ISSUE #10: [BUG] version command doesn't work
# Bug Description
`kube-burner-ocp version` errors out if prometheus cannot be found.

## **Output of `kube-burner-ocp version`**
```
$  kube-burner-ocp version
time="2024-02-06 12:47:43" level=fatal msg="error obtaining Prometheus information: Unauthorized" file="ocp.go:95"
```

## **Describe the bug**

`version` command tries to fetch prometheus information.

## **To Reproduce**

1. Download and extract kube-burner-ocp v1.1.0 tarball
1. `kube-burner-ocp version`

## **Expected behavior**

Output version without attempting to fetch prometheus information.

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Skipping PersistentPreRun() when using the version subcmd

- Related Issue #
- Closes #10

### CODE REVIEW

File: cmd/ocp.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ func openShiftCmd() *cobra.Command {
 	ocpCmd.MarkFlagsRequiredTogether("es-server", "es-index")
 	ocpCmd.MarkFlagsMutuallyExclusive("es-server", "local-indexing")
 	ocpCmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		if cmd.Name() == "version" {
```
Comment (vishnuchalla) [inline cmd/ocp.go:66]: I am bit confused on where this issue is actually happening. When I try to reproduce in my local, I do not see any issues.
```
[vchalla@vchalla-thinkpadp1gen2 kube-burner-ocp]$ kube-burner-ocp version
Version: main
Git Commit: dc8ff253f51b379fb63fe703e263ffc3e184e1d6
Build Date: 2024-02-14-09:50:57
Go Version: go1.21.5
OS/Arch: linux amd64
```

Comment (rsevilla87) [reply cmd/ocp.go:66]: The issue is that when you run `kube-burner-ocp version` it tries to discover the prometheus endpoint + token. For example, it fails when kubeconfig is not configured

**Comment by rsevilla87** [comment]
@afcollins PTAL

**Review by afcollins — APPROVED** [review]
LGTM!

It works:
```
$ ./bin/amd64/kube-burner-ocp version
Version: issue-10
Git Commit: c35b5bf06349de3b053e9dc4ce8aec2f4b0ce6d1
Build Date: 2024-02-15-11:41:21
Go Version: go1.19.4
OS/Arch: darwin amd64
```

Thank you!

**Pull request merged**